feature-profile/src/main/java/cm/aptoide/pt/profile/presentation/ProfileView.kt

```diff
@@ -399,34 +394,34 @@ private fun ProfileCard(
       .fillMaxWidth()
       .padding(horizontal = 16.dp)
       .clickable(onClick = onClick),
-    elevation = 10.dp,
-    shape = RoundedCornerShape(20.dp)
+    elevation = 2.dp,
+    shape = shapes.large
```

Comment:
Also change profile card background color to match the one in design

Reply:
I have the impression that the profile card background color is already matching the one in design, because, by default, card use the `surface` field in `Material` in which the `Theme.kt` class is already specifying to be the colors in design